### ISSUE TITLE
Screen Reader is not announcing associated information.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13996]
**Test Environment**
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)
URL: data:text/html,%3cbutton%3eoutter%20target1%3c/button%3e%3cnav%20aria-labelledby=%22nav-1%22%3e%3ch2%20id=%22nav-1%22%3eTest%20navigation%3c/h2%3e%3cbutton%3einner%20target%3c/button%3e%3c/nav%3e

**Repro Steps:**
1. Visit this URL data:text/html,%3cbutton%3eoutter%20target1%3c/button%3e%3cnav%20aria-labelledby=%22nav-1%22%3e%3ch2%20id=%22nav-1%22%3eTest%20navigation%3c/h2%3e%3cbutton%3einner%20target%3c/button%3e%3c/nav%3e

https://user-images.githubusercontent.com/93735775/183353955-e3dbf693-9827-401e-a28d-e579588c7d4a.mp4

2. Hit the b key to focus the button in the region. arrowing also works but the button skips the header.
3. Observe what NVDA syas.
4. Use object navigation to escape the region.

**Actual Behavior**
Upon entering, nvda does not read the labelledby but it does read the labelledby when I touch the node with object nav.

**Expected Behavior**
The heading is read.

**Comment by alexarnaud** [comment]
Hello,

What I understand from your bug report is: you would like NVDA to announce it entered into a "nav" section with the label attached to the nav section, isn't it?

If yes, please update your bug accordingly to make it clearer.

**Comment by seanbudd** [comment]
Closing as duplicate of #13307

**Comment by seanbudd** [comment]
Note: this video appears to be using narrator
